### ISSUE TITLE
Images not displayed though compilation is successful

### ISSUE THREAD

**Issue by LeBarbouze** [readthedocs/readthedocs.org#6065]
## Details

* Read the Docs project URL: https://readthedocs.org/projects/tunacell/
* Build URL (if applicable): https://readthedocs.org/projects/tunacell/builds/9497770/
* Read the Docs username (if applicable): LeBarbouze

## Expected Result

I expect the images to be actually displayed, for instance here https://tunacell.readthedocs.io/en/latest/users/tutorial.html#calling-the-plotting-function (scroll at the bottom of the subsection).

This is actually what I get with a local build on my computer (same sphinx version 1.8.5 than in the compilation at readthedocs).

## Actual Result

The browser only shows up the (relative) path as a link, and inspection shows that the image "cannot be displayed". The link actually works (it's not a 404 problem). The browser warns the image contains errors. However the PNG image can be read from the GitHub repo, see for instance https://github.com/LeBarbouze/tunacell/blob/master/docs/images/tutorial_sample-plot.png.

The compilation is marked "successful", and the logs indicate the images have been copied.

**Comment by stsewd** [comment]
Can you link to a page where you expect the image and isn't there? Also, take into consideration that images should have a relative path, since when you host  your docs on rtd, the pages are in `lang/version/`

**Comment by LeBarbouze** [comment]
Thanks for your quick reply.

Here's a link of a subsection where the image should be: https://tunacell.readthedocs.io/en/latest/users/tutorial.html#calling-the-plotting-function (scroll at the bottom of the subsection). I tried the relative path in the source document for this image. If you scroll down, there are a couple of images where I put the absolute path from the root directory (docs/ in the repo). The link is actually present, so I am guessing that paths are alright. However when clicked I get a message saying that the image cannot be displayed because it contains errors, which is not the case locally, and the PNG image is correctly displayed from the Github repo, see https://github.com/LeBarbouze/tunacell/blob/master/docs/images/tutorial_sample-plot.png. I mention that these images are git using git LFS, no clue if it's helpful or not. In the compilation logs, the image are correctly copied, so I don't get what happens.

**Comment by LeBarbouze** [comment]
@stsewd I've added links in the comment above, and also edited the issue description. Thanks for pointing to those details.

**Comment by stsewd** [comment]
>  I mention that these images are git using git LFS, no clue if it's helpful or not. In the compilation logs, the image are correctly copied, so I don't get what happens.

I think maybe that's the problem. I don't have experience with git lfs, but I've seen this #1846

**Comment by humitos** [comment]
> I mention that these images are git using git LFS, no clue if it's helpful or not. In the compilation logs, the image are correctly copied, so I don't get what happens.

This is definitely the issue. I wrote a workaround some time ago to make it work under Read the Docs: https://github.com/readthedocs/readthedocs.org/issues/1846#issuecomment-477184259

Keep in mind that this is not officially supported by Read the Docs.

That said, let us know if you were able to make it work.

**Comment by LeBarbouze** [comment]
Hello guys @stsewd @humitos, thanks for your kind help. I implemented one of the workarounds proposed in #1846 and it works fine. I used the one with the Python git_lfs module. @humitos's reply with  calling the OS to download the git LFS client seems to be a valid solution as well, without the need to rely on another Python module.

I guess the incentive to implement Git LFS compatibility in ReadTheDocs is only growing in the community of RTD users. All the best with this piece of work if you guys get to it!
